code_examples/sdk_examples/src/core_features/messaging/index.ts

```diff
@@ -0,0 +1,72 @@
+import * as Kilt from '@kiltprotocol/sdk-js'
+import { createCompleteFullDid } from '../did/05_full_did_complete'
+import { createDriversLicenseCType } from '../claiming/01_create_ctype'
+import { decryptMessage } from './03_decrypt_message'
+import { encryptMessage } from './02_encrypt_message'
+import generateKeypairs from '../utils/generateKeypairs'
+import { generateMessage } from './01_generate_message'
+
+// Runs through the messaging encryption and decryption of messages
+export async function runAll(submitterAccount: Kilt.KiltKeyringPair) {
+  console.log('Running messaging flow...')
+  console.log('Generating a sender DID')
```

Comment:
Because these flows are run in parallel, these console logs should be namespaced with something like we do for the other flows. E.g., `console.log(1 Messaging) Generating the sender's DID)` and so on.

Reply:
I can do.